idpf_specification.md

```diff
@@ -11541,3 +11520,133 @@ union virtchnl2_rx_desc {
 
 #endif /* _VIRTCHNL_LAN_DESC_H_ */
 ```
+# Appendix : Linux RDMA
+
+## North Side Interfaces and flows for Linux: 
+This section is not part of the main spec and it is just an example implementation detail referred from the Linux support for RDMA driver.
+
+###	RDMA Aux drivers naming
+IDPF driver will create a Core and multiple vPort level Aux device instances for RDMA. The Core RDMA device driver will setup the Control Queues with RDMA FW and any other Interface level initialization and the Vport Aux device instance can be one to one with the LAN vport netdev instances. Although not ever Lan vPort netdev may not create a RDMA vport Ib driver instance as discussed earlier. 
+( Note: TBD, the LAN driver itself is also undergoing changes to create LAN netdevs as Aux device instances on the aux bus. Although RDMA vPort Aux device instances are not dependent on the LAN Aux Devices per Netdev and so the work can be done independently)  
+RDMA Aux devices will be named carrying the following key words in the name: IDPF, VENDOR ID, RDMA, , TYPE OF the DEVICE (CORE or VPort), A Global ID 
+This allows for a unique vendor specific RDMA driver to be loaded along with IDPF driver.
+The Global unique ID, is requested from the OS and helps identify individual Aux driver instances.
+###	RDMA Driver bring up and teardown flows
```

Comment:
bring-up is called out in this section but there is little mention of the tear-down flow.